src/rules/order.js

```diff
@@ -244,16 +244,37 @@ function getSorter(ascending) {
   const multiplier = ascending ? 1 : -1;
 
   return function importsSorter(importA, importB) {
-    let result;
-
-    if (importA < importB) {
-      result = -1;
-    } else if (importA > importB) {
-      result = 1;
+    let result = 0;
+
+    if (!importA.includes('/') && !importB.includes('/')) {
+      if (importA < importB) {
+        result = -1;
+      } else if (importA > importB) {
+        result = 1;
+      } else {
+        result = 0;
+      }
     } else {
-      result = 0;
-    }
+      const A = importA.split('/');
+      const B = importB.split('/');
+      const a = A.length;
+      const b = B.length;
+
+      for (let i = 0; i < Math.min(a,b); i++) {
+        if (A[i] < B[i]) {
+          result = -1;
+          break;
+        } else if (A[i] > B[i]) {
+          result = 1;
+          break;
+        }
```

Comment:
var result = A[i] - B[i];
if (result) {
  break;
}

Reply:
But we are returning the result, and then its value can be something other than -1/0/1. Will that be fine?

Reply:
Yep! As long as it's a number - zero, negative, or positive - it's a valid comparator.

Reply:
My bad, even A[i] and B[i] are strings.

Reply:
Per discussion, we'll need to keep greater/less comparison here.